app/mailers/subscription_mailer.rb

```diff
@@ -1,16 +1,16 @@
 class SubscriptionMailer < ApplicationMailer
   def confirmation(subscription_id)
-    subscription = Subscription.find(subscription_id)
+    @subscription = Subscription.find(subscription_id)
 
-    @subscription_reference = subscription.reference
-    @search_criteria = SubscriptionPresenter.new(subscription).filtered_search_criteria
-    @expires_on = subscription.expires_on
-    @unsubscribe_token = subscription.token
+    @subscription_reference = @subscription.reference
+    @search_criteria = SubscriptionPresenter.new(@subscription).filtered_search_criteria
```

Comment:
If we pass the presenter to the views, can we make the view responsible for getting the `filtered_search_criteria`?